src/index.js

```diff
@@ -305,10 +305,10 @@ class List {
     const items = this._elements.wrapper.querySelectorAll(`.${this.CSS.item}`);
 
     for (let i = 0; i < items.length; i++) {
-      const value = items[i].innerHTML.replace('<br>', ' ').trim();
+      const value = items[i].innerHTML.replace(/<br>|&nbsp;/gi, ' ').trim();
```

Comment:
Do not worry about inline style list for list items i was kept it in my mind so i has been test it and i didn't found any problems